_posts/2022-01-25-Related-Projects.md

```diff
@@ -73,4 +73,15 @@ Koord adds real-time video of participants to Jamulus technology in a new applic
 
 The [Melomax website](https://melomax.live) offers private servers groups can use with Jamulus client software.
 
+## Running a Jamulus server on a Raspberry Pi and how to measure the transmission quality on Jamulus
+
+This project describes how to use a Raspberry Pi as a Jamulus client.
+It also examines the influence of the Jamulus settings on the transmission quality and how this can be measured. This was first tested with a function generator and an oscilloscope. A device was then developed with which the transmission errors can be measured easily and automatically. Circuit diagrams, layout and software for this device are described and made available.
+The documentation is partly in German and partly in English.
+
+I hope that some Jamulus users will find our project helpful and interesting. Therefore I would like to make it generally available.
```

Comment:
```suggestion
```

I'd rather not add an "I" here since the author is not visible by default on this page.